### ISSUE TITLE
Don't warn about icon size mismatch if icon is SVG

### ISSUE THREAD

**Issue by asamuzaK** [mozilla/addons-linter#1669]
### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)
When I uploaded add-on to AMO, it is warned about icon size mismatch.

```
The size of the icon does not match the manifest.

警告: Expected icon at "icon.svg" to be 32 pixels wide but was 128.
```

My manifest.json is like this.
```
  "icons": {
    "32": "icon.svg#gray",
    "64": "icon.svg"
  },
```

It is an SVG file, so size should not matter.

### What happened?
AMO raised warning.

### What did you expect to happen?
No warning.

### Anything else we should know?
(Please include screenshots and any relevant files.)

In [icons - Mozilla | MDN](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/icons "icons - Mozilla | MDN"), it is explained as:

> Even though you can use one file, you still need to specify various size of the icon in your manifest. E.g.:
```
"icons": {
  "48": "icon.svg",
  "96": "icon.svg"
}
```

**Comment by erosman** [comment]
Was this one filled and pushed? 
The error is still present.

**Comment by asamuzaK** [comment]
In addons-linter, yes.
I don't know it is pushed to AMO server.

**Comment by asamuzaK** [comment]
Latest release seems to be [Release 2017.12.07 · mozilla/addons-server](https://github.com/mozilla/addons-server/releases/tag/2017.12.07 "Release 2017.12.07 · mozilla/addons-server") but linter is merged on Dec 08, 2017? [Comparing 2017.12.07...master · mozilla/addons-server](https://github.com/mozilla/addons-server/compare/2017.12.07...master "Comparing 2017.12.07...master · mozilla/addons-server")

**Comment by asamuzaK** [comment]
Confirmed fixed on AMO server too.

**Comment by AlexandraMoga** [comment]
Tried validating an extension which included `.svg` icons with wrong sizes defined in the manifest and no errors/warnings  were thrown. 
Marking this as verified fixed.

**Comment by erosman** [comment]
There are still warnings ....

`https://reviewers.addons.mozilla.org/en-US/firefox/files/browse/817497/`

> Warning: The size of the icon does not match the manifest.
> Expected icon at "images/rikaichamp-blue.svg" to be 48 pixels wide but was 200.
> 
> Warning: The size of the icon does not match the manifest.
> Expected icon at "images/rikaichamp-blue.svg" to be 96 pixels wide but was 200.
> 
> Warning: The size of the icon does not match the manifest.
> Expected icon at "images/rikaichamp-blue.svg" to be 128 pixels wide but was 200.

**Comment by asamuzaK** [comment]
> There are still warnings ....
> 
> `https://reviewers.addons.mozilla.org/en-US/firefox/files/browse/817497/`

I think you should file a new issue in [mozilla/addons-server](https://github.com/mozilla/addons-server "mozilla/addons-server: 🕶 addons.mozilla.org Django app and API 🎉") for that?
